File: bmi/bmiimperial.py
# For example, an execution could look like this:

#    Please enter weight in kilograms: 50
#    Please enter height in meters: 1.58
#    BMI is: 20.0288415318

weight = float(input("Please enter weight in pounds: "))
height = float(input("Please enter height in inches: "))
print(f"BMI is: {weight * 0.453592 / ((height * 0.0254) ** 2)}")

File: bmi/bmiimperial.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main()
{
    int weight;
    float height;
    cin >> weight;
    cin >> height;
    cout << "bmi is: " << round(weight * 0.453592 / pow(height * 0.0254, 2) * 100.0) / 100.0 << endl;
    return 0;
}